docs/visual-basic/programming-guide/language-features/arrays/how-to-sort-an-array.md

```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: How to: sort an array in Visual Basic"
```

Comment:
Why does this one get "in Visual Basic" while others get (Visual Basic)?

Reply:
Never mind, I see now that the text is coming from the H1, not the metadata title.